coap-gateway/coap-gw.yaml

```diff
@@ -0,0 +1,68 @@
+log:
+  debug: true
+apis:
+  coap:
+    address: "0.0.0.0:5684"
+    external-port: 5684
+    fqdn: "localhost"
```

Comment:
these 2 settings we want to replace by `externalAddress` -> it is in format `fqdn:external-port`

Reply:
You means you want to merge two settings "external-port" and "fqdn" with one setting "externalAddress" ? Sure to remove fqdn ?

Reply:
yes